lib/logstash/outputs/elasticsearch.rb

```diff
@@ -368,9 +370,31 @@ def submit(actions)
   def flush(actions, teardown = false)
     begin
       submit(actions)
+    rescue Manticore::SocketException => e
+      # If we can't even connect to the server let's just print out the URL (:host is actually a URL)
+      # and let the user sort it out from there
+      @logger.error(
+        "Attempted to send a bulk request to Elasticsearch configured at '#{@current_client.client_options[:host]}',"+
+          " but Elasticsearch appears to be unreachable or down!",
+        :client_config => @current_client.client_options,
+        :error_message => e.message,
+        :error_class => e.class.name,
+        :backtrace => e.backtrace
```

Comment:
I'm hesitant to show the backtrace by default, especially in an error message. My tendency is to hide backtraces because they confuse users and make helping them harder (anecdotally).

I'm open to including it here, but tell me a good story about it? :)